cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,32 @@
-describe('', () => {
-  before(() => {
+import ArticlePageObject from '../support/pages/article.pageObject';
 
-  });
+const ArticlePage = new ArticlePageObject();
 
-  it('', () => {
+describe('Article', () => {
+  let user;
 
+  beforeEach(() => {
+    cy.task('generateUser').then((generatedUser) => {
+      user = generatedUser;
+      cy.login(user.email, user.username, user.password);
+      cy.visit('/editor');
+    });
   });
 
-  it('', () => {
+  it('should fill in article and create it', () => {
+    const article = ArticlePage.generateArticle();
+    ArticlePage.createArticle(article);
+    ArticlePage.onClickSubmitArticle();
+    ArticlePage.assertArticleTitle(article.title);
+  });
 
+  it('should be able to delete an article', () => {
+    const article = ArticlePage.generateArticle();
+    cy.createArticle(article).then((response) => {
```

Comment:
Ensure that the `cy.createArticle` command is correctly defined and returns a response with the article slug. This is crucial for navigating to the article page.